packages/suite/src/components/wallet/CoinmarketTag/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import { variables } from '@trezor/components';
+
+const TagRow = styled.div`
+    display: flex;
+`;
+
+const Tag = styled.div`
+    margin: 0 16px 2px 16px;
+    padding: 0 8px;
+    border-radius: 8px;
+    background: ${props => props.theme.TYPE_ORANGE};
+    color: ${props => props.theme.TYPE_WHITE};
+    font-size: ${variables.FONT_SIZE.TINY};
+    line-height: 21px;
+    text-transform: uppercase;
+`;
+
+interface Props {
+    tag?: string;
+}
+
+const CoinmarketTag = ({ tag }: Props) => <TagRow>{tag && <Tag>{tag}</Tag>}</TagRow>;
+
+export default CoinmarketTag;
```

Comment:
We want to stop using `export default` and `index.tsx` files, reasons explained [in internal Notion wiki ](https://www.notion.so/satoshilabs/Code-Improvements-1-Code-style-changes-0f1bb44d86764715b8f5962c7f23466e). 

Despite the temporary inconsistency – there is no other way to get rid of it.

Our usual approach is to create separate followup PR with just refactoring of the touched part. I understand that there's a lot of components here to refactor, so I would suggest to stop using it at least in this new component. What do you think?